contracts/whitelists/whitelist-immutable-flex/src/state.rs

```diff
@@ -0,0 +1,21 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, Decimal, Uint128};
+use cw_storage_plus::{Item, Map};
+
+#[cw_serde]
+pub struct Config {
+    pub admin: Addr,
+    pub mint_discount_bps: Option<u64>,
+}
+
+impl Config {
```

Comment:
Why do we need a whitelist wide discount bps?

Reply:
It's an optional parameter and we can remove it, but my intention was not to take away from the original contract in case it's used again for its initial purpose but with a `mint_count` for each address.